collect_app/src/main/java/org/odk/collect/android/formentry/FormEntryViewModel.java

```diff
@@ -174,36 +172,26 @@ public void openHierarchy() {
     }
 
     public void logFormEvent(String event) {
-        analytics.logFormEvent(event, getFormIdentifierHash());
+        AnalyticsUtils.logFormEvent(event);
     }
 
     public NonNullLiveData<Boolean> hasBackgroundRecording() {
         return hasBackgroundRecording;
     }
 
-    private String getFormIdentifierHash() {
-        if (formController != null) {
-            return formController.getCurrentFormIdentifierHash();
```

Comment:
I think this method in `FormController` is no longer used.

Reply:
Ah yeah good spot